setup.py

```diff
@@ -98,7 +98,7 @@ def read_deps_from_file(filname):
     ],
     long_description=long_description,
     long_description_content_type="text/markdown",
-    python_requires=">=3.9",
+    python_requires=">=3.10",
```

Comment:
Avoids the need for future updates
```suggestion
    python_requires=f">={REQUIRED_MAJOR}.{REQUIRED_MINOR}",
```